spec/views/dashboard/student_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'dashboard/student.haml' do
+  context 'interview is present' do
+    before :each do
+      @interview = create :interview
+      assign :interviews, Array(@interview)
+    end
+    it 'displays the date, time, and location of each interview' do
+      render
+      expect(rendered).to include @interview.information
+    end
+    it 'contains a link to review the application' do
+      render
+      action_path = application_record_path @interview.application_record
+      expect(rendered).to have_tag '.interview_review_link' do
```

Comment:
Since your test looks for a div with a specific CSS class, I'd like to see that reflected in the test description.